src/coffee/services.coffee

```diff
@@ -304,6 +304,48 @@ define (require, exports, module) ->
 
   Services.service 'WebServiceAuth', ['apiTokenPromise', '$rootScope', 'authorizer', WebServiceAuth]
 
+  Services.factory 'Projects', Array '$http', '$log', 'WebServiceAuth', (http, logger, auth) ->
```

Comment:
This file has got way too large - we both need to refactor it into smaller units.